library/ZendService/Twitter/Twitter.php

```diff
@@ -318,7 +336,8 @@ public function accountVerifyCredentials()
     {
         $this->init();
         $response = $this->get('account/verify_credentials');
-        return new Response($response);
+        $this->twitterResponse->populate($response);
```

Comment:
A response is a value object, that doesn't need replacement via a mock.

In this scenario, a proper test would do something like following (more assertions on the request needed):

```php
public function testAccountVerifyCredentials()
{
    $response = new HttpResponse();
    $response->setBody('the actual json here');
    $client = $this->createMock(HttpClient::class);
    $client->expects(self::once())->method('send')->with(self::isInstanceOf(Request::class))->willReturn($response);
    self::assertEquals(new Response($response), $this->twitter->accountVerifyCredentials());
}
```